### DISCUSSION TITLE
Need to update Path and Body parameters DOC default values examples

### DISCUSSION THREAD

**Discussion by akash-vns** [fastapi/fastapi#6399] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from typing import Union

from fastapi import Body, FastAPI, Path
from pydantic import BaseModel, Field

app = FastAPI()


class Item(BaseModel):
    name: str
    description: Union[str, None] = Field(
        default=None, title="The description of the item", max_length=300
    )
    price: float = Field(gt=0, description="The price must be greater than zero")
    tax: Union[float, None] = None


@app.post("/{data_id:str}")
async def index(data_id: int = Path(title="The ID of the item to get", ge=0, le=1000),
                data: Item = Body(embed=True)):
    if data:
        print(data)
    return data
```

### Description

Hello Team, 

When i try to follow the same steps that are described into the ([Body-Fields](https://fastapi.tiangolo.com/tutorial/body-fields/) Doc , and [Path-params](https://fastapi.tiangolo.com/tutorial/path-params-numeric-validations/) Doc )
and its give us default values are required error .

(when i try to run above code and that code gave me a default value  is required error )

Document is not have update that default values is required into the Path and Body like Query

happy to raise PR for the Doc update.

### Operating System

Linux

### FastAPI Version

0.77.1

### Python Version

3.8

**Comment by aasiffaizal** [comment, marked as answer]
Hi @akash-vns 
I think that is because of the new release [0.78.0](https://github.com/tiangolo/fastapi/releases/tag/0.78.0)
`...` has been omitted as the default value for `Path`, `Body`, etc in the new version. Check this [PR](https://github.com/tiangolo/fastapi/pull/4906) out. The PR that you have raised is reverting the docs to the older version.

**Comment by akash-vns** [comment]
Okay thank you so much @aasiffaizal 
will close the PR and issue ,
